libs/sdk-bindings/bindings-swift/Package.swift

```diff
@@ -7,16 +7,15 @@ let package = Package(
     name: "bindings-swift",
     platforms: [
         .macOS(.v12),
-        .iOS(.v11),
+        .iOS(.v14),
```

Comment:
Sorry I forgot this. Can you change this also back to v11?

Reply:
b60d13ad9375f7e451bed5290a66c28ff2f29014